examples/pipelines/simple_pipeline/components/write_to_hub/src/main.py

```diff
@@ -0,0 +1,57 @@
+"""
+This component write an images dataset to the hub.
+"""
+import logging
+
+import huggingface_hub
+import dask.dataframe as dd
+
+from fondant.component import FondantTransformComponent
+from fondant.logger import configure_logging
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+class WriteToHubComponent(FondantTransformComponent):
+    def transform(
+        self,
+        *,
+        dataframe: dd.DataFrame,
```

Comment:
```suggestion
        dataframe: dd.DataFrame,
        *,
```
It might be that you'll need to change this to this to run the component successfully

Reply:
I think it should be fine since the FondantTransformComponent is currently defined like this 
  ```
  def transform(self, *args, **kwargs) -> dd.DataFrame
  ```